Meadowlark-js/backends/meadowlark-postgres-backend/src/security/SecurityResponse.ts

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Thinking down the road, once we get PostgreSQL running, we should look for commonality with the MongoDB backend to extract to a library, maybe named "meadowlark-backend-support"

Reply:
agreed